apps/web/src/utils/date.ts

```diff
@@ -100,6 +100,11 @@ export function isBetween(startDate: Date, endDate: Date): boolean {
   return isPast(startDate) && isFuture(endDate);
 }
 
+export function hoursBetween(startDate: Date | null, endDate: Date | null): number {
+  if (!startDate || !endDate) return 0;
+  return Math.abs(endDate.getTime() - startDate.getTime()) / 36e5;
+}
```

Comment:
Bruke `date-fns` sin `differenceInHours`? https://date-fns.org/v3.6.0/docs/differenceInHours

Om du vil ha guards for `null` lage `_differenceInHours` som en liten wrapper.

```suggestion
import { differenceInHours } from "date-fns";

// ...

export function _differenceInHours(startDate: Date | null, endDate: Date | null): number {
  if (!startDate || !endDate) return 0;
  return differenceInHours(startDate, endDate);
}
```